curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-the-color-of-an-svg-element.md

```diff
@@ -14,7 +14,7 @@ In SVG, a `rect` shape is colored with the `fill` attribute. It supports hex cod
 
 # --instructions--
 
-Add an `attr()` method to set the "fill" of all the bars to the color "navy".
+Add an `attr()` method to set the `fill` of all the bars to the color `navy`.
```

Comment:
Here, `navy` is within backticks, but the names of the colors are not, in the `hints` section.

Reply:
So this is actually a good question.

The instructions read like we expect the camper to use "color: navy", but the test checks the actual rgb value so it's not 100% necessary to use `navy`. I could go either way on this one.

Reply:
If the user specifies the color "navy" like `.attr("fill", "navy");` or `rgb(0, 0, 128)`, will the test pass?  If so, then I would remove the backticks and just use the word **navy** (not necessarily bold).

Reply:
Correct - it passes if I use `.attr("fill", "rgb(0,0,128)")`.